languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt

```diff
@@ -9193,4 +9193,11 @@ Verne	Verne	NPMS000
 chifruda	chifrudo	NCFS000
 chifrudas	chifrudo	NCFP000
 tórax	tórax	NCMN000
-
+homeostase	homeostase	NCFS000
+homeostases	homeostase	NCFP000
+crónico	crónico	AQ0MS0
+crónicos	crónico	AQ0MP0
```

Comment:
The pt-PT speller already recognizes `crónico` and its variants. It'd be best not to add those to the general `pt` speller, as this spelling is incorrect for pt-BR.